src/components/com_tjdashboard/administrator/tables/widgets.php

```diff
@@ -8,6 +8,7 @@
  */
 
 defined('_JEXEC') or die;
+JLoader::import('components.com_tjdashboard.includes.widget', JPATH_ADMINISTRATOR);
```

Comment:
@shitalkakade24 This is not required. The dependencies must be bootstrapped.

Reply:
removed